app/controllers/krikri/reports_controller.rb

```diff
@@ -0,0 +1,9 @@
+module Krikri
+  class ReportsController < ApplicationController
+
```

Comment:
I think this controller also needs `layout 'krikri/application'`, which seems to fix the issue I reported. (There may be a more exhaustive way to fix that.)